src/views/open-api-docs-view-v2/server.ts

```diff
@@ -3,138 +3,314 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+// Utils
+import { cachedGetProductData } from 'lib/get-product-data'
+import { fetchCloudApiVersionData } from 'lib/api-docs'
+import { getVersionSwitcherProps } from 'views/open-api-docs-view/utils'
+import { isDeployPreview } from 'lib/env-checks'
 import { parseAndValidateOpenApiSchema } from 'lib/api-docs/parse-and-validate-open-api-schema'
+import { serialize } from 'lib/next-mdx-remote/serialize'
 import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+import fetchGithubFile from 'lib/fetch-github-file'
 import isAbsoluteUrl from 'lib/is-absolute-url'
-// Utils
+// Utils, local
+import { findDefaultVersion } from './utils/find-default-version'
 import getOperationContentProps from './components/operation-content/server'
-import { serialize } from 'lib/next-mdx-remote/serialize'
 import {
 	getOperationObjects,
 	OperationObject,
 } from './utils/get-operation-objects'
-import { wordBreakCamelCase } from './utils/word-break-camel-case'
 import { groupItemsByKey } from './utils/group-items-by-key'
-import { cachedGetProductData } from 'lib/get-product-data'
+import { slugifyOperationId } from './utils/slugify-operation-id'
+import { wordBreakCamelCase } from './utils/word-break-camel-case'
+import { parseOpenApiV2UrlContext } from './utils/parse-open-api-v2-url-context'
 // Types
+import type { ApiDocsVersionData } from 'lib/api-docs/types'
+import type { BreadcrumbLink } from '@components/breadcrumb-bar'
+import type { GetStaticPaths } from 'next'
+import type { GithubDir } from 'lib/fetch-github-file-tree'
 import type {
+	ApiDocsUrlContext,
+	OpenApiDocsV2Params,
+	OpenApiDocsViewV2Config,
 	OpenApiDocsViewV2Props,
 	SharedProps,
-	OpenApiDocsViewV2Config,
 } from 'views/open-api-docs-view-v2/types'
-import { OpenAPIV3 } from 'openapi-types'
+import type { OpenAPIV3 } from 'openapi-types'
+
+/**
+ * Generate static paths for an OpenAPI docs view.
+ */
+export async function generateStaticPaths({
+	schemaSource,
```

Comment:
This file includes a few major changes.

First, `generateStaticPaths` now expects a `schemaSource`, which we use to fetch and parse which operations are in the "default version" of the schema (ie the version that will be rendered at non-versioned URL paths). This allows us to statically render individual operation pages.